automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/PublishSpec.scala

```diff
@@ -18,73 +18,30 @@ import scala.concurrent.duration.DurationLong
 
   implicit val ec: ExecutionContextExecutor = ExecutionContext.global
-  implicit override val patienceConfig = PatienceConfig(timeout = scaled(Span(10, Seconds)), interval = scaled(Span(500, Millis)))
+  implicit override val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(10, Seconds)), interval = scaled(Span(500, Millis)))
 
-  val autocompleteTextQueryPrefix: String = "cance"  // partial string of "cancer" to test autocomplete
-  val minNumOfResults: Int = 5
 
   "For a user with publish permissions" - {
-    "an unpublished workspace" - {
-      "without required library attributes" - {
-        "publish button should be visible but should open error modal when clicked" in {
-          val curatorUser = UserPool.chooseCurator
-          implicit val curatorAuthToken: AuthToken = curatorUser.makeAuthToken()
-          withCleanBillingProject(curatorUser) { billingProject =>
-            withWorkspace(billingProject, "PublishSpec_curator_unpub_") { wsName =>
-
-              withWebDriver { implicit driver =>
-                withSignIn(curatorUser) { _ =>
-                  val page = new WorkspaceSummaryPage(billingProject, wsName).open
-                  val messageModal = page.clickPublishButton(expectSuccess = false)
-                  eventually { messageModal.isVisible shouldBe true }
-                  messageModal.clickOk()
-                }
-              }
-            }
-          }
-        }
-      }
-      "with required library attributes" - {
-        "publish button should be visible " in {
-          val curatorUser = UserPool.chooseCurator
-          implicit val curatorAuthToken: AuthToken = curatorUser.makeAuthToken()
-          withCleanBillingProject(curatorUser) { billingProject =>
-            withWorkspace(billingProject, "PublishSpec_curator_unpub_withAttributes_") { wsName =>
-              api.library.setLibraryAttributes(billingProject, wsName, LibraryData.metadataBasic)
-
-              withWebDriver { implicit driver =>
-                withSignIn(curatorUser) { wsList =>
-                  val page = new WorkspaceSummaryPage(billingProject, wsName).open
-                  eventually { page.hasPublishButton shouldBe true }
-                }
-              }
-            }
-          }
-        }
-      }
-    }
     "a published workspace" - {
       "should be visible in the library table" taggedAs Tags.SmokeTest in {
         val curatorUser = UserPool.chooseCurator
         implicit val curatorAuthToken: AuthToken = curatorUser.makeAuthToken()
         withCleanBillingProject(curatorUser) { billingProject =>
           withWorkspace(billingProject, "PublishSpec_curator_publish_") { wsName =>
-            withCleanUp {
-              val data = LibraryData.metadataBasic + ("library:datasetName" -> wsName)
-              api.library.setLibraryAttributes(billingProject, wsName, data)
-              register cleanUp api.library.unpublishWorkspace(billingProject, wsName)
```

Comment:
`register cleanup unpulishWorkspace` not necessary because `workspace delete` automatically unpublish workspace.